# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoader.cpp
# Commit: 0a3d9301244b
# Full Hash: 0a3d9301244b91559c4c19314ce991525d9f605c
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mElement from ScriptFetchOptions to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136742
# ==============================================================================

diff -r 46d45c7bae2f -r 0a3d9301244b dom/script/ModuleLoader.cpp
--- a/dom/script/ModuleLoader.cpp	Fri Feb 18 12:45:48 2022 +0000
+++ b/dom/script/ModuleLoader.cpp	Fri Feb 18 12:45:49 2022 +0000
@@ -267,7 +267,8 @@
     }
 
     RefPtr<ModuleScript> moduleScript =
-        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL);
+        new ModuleScript(aRequest->mFetchOptions, aRequest->mBaseURL,
+                         aRequest->mLoadContext->mElement);
     aRequest->mModuleScript = moduleScript;
 
     if (!module) {